Tests/Unit/SecurityManagerTest.php

```diff
@@ -150,6 +150,10 @@ public function testValidateValidRequest()
         $this->assertTrue($this->securityManager->validateRequest($this->commandRequest));
     }
 
+    /**
+     * @expectedException \Exception
```

Comment:
Once you throw exceptions with a defined exception code I would prefer of asserting that one as the messages might change easily.